lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -878,6 +878,16 @@ void OpenCLFunction::execute() {
       setKernelArg(kernel, numArgs + 5, idim);
       setKernelArg(kernel, numArgs + 6,
                    ShapeNHWC(CC->getFilter()->getType()->dims()));
+      if (isQuantized) {
```

Comment:
you need to enable isOpSupported for the convolution node.

Reply:
this is somewhat unrelated... Since you added Max support to openCL you should enable ReLU as well as it's lowered to the Max node.

Reply:
cool. Will do after OCLConvolution

Reply:
sounds good, that will be one line change :)